integreat_cms/cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html

```diff
@@ -7,7 +7,7 @@
     {% url 'pages' region_slug=request.region.slug language_slug=default_language_slug %}?status=REVIEW
 {% endblock todo_dashboard_title_link %}
 {% block todo_dashboard_title %}
-    {% translate "Pages that are waiting for review" %}
+    {% translate "Pages pending approval" %}
```

Comment:
What about including "with" when talking about pages? I'm not a native English speaker and I can't explain my gut feeling, but this sounds a bit more natural to me:

```suggestion
    {% translate "Pages with pending approval" %}
```